project/plugins.sbt

```diff
@@ -1,4 +1,5 @@
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0-M2")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.3.9")
+//addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0-M2")
```

Comment:
Sorry for the late review. I wasn't sure about this change.

Reply:
Goodness 😝